nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -778,6 +767,29 @@ def test_plot_surf_roi_matplotlib_specific():
     plt.close()
 
 
+def test_plot_surf_roi_matplotlib_specific_plot_to_axes():
+    """Test plotting directly on some axes."""
+    mesh, roi_map, _ = _generate_data_test_surf_roi()
+
+    plot_surf_roi(mesh, roi_map=roi_map, axes=None,
+                  figure=plt.gcf(), engine='matplotlib')
+
+    _, ax = plt.subplots(subplot_kw={'projection': '3d'})
```

Comment:
extracted those tests into a separate one

`ax=plt.gca()` was leading to test failures